djangoevents/settings.py

```diff
@@ -0,0 +1,13 @@
+from django.conf import settings
+
+
+_DEFAULTS = {
+    'EVENT_SCHEMA_VALIDATION': {
+        'VALIDATOR': 'djangoevents.schema.LocalRepositoryValidator',
+        'VALIDATOR_SCHEMA_DIR': 'avro',
+    },
+}
+
+# library user must be able to override the defaults.
+# TODO: settings validation
+CONFIG = getattr(settings, 'EVENTSOURCING', _DEFAULTS)
```

Comment:
As proposed in https://github.com/ApplauseOSS/djangoevents/issues/19 let's keep the configuration named `DJANGOEVENTS_CONFIG`